ts/components/conversation/composition/CompositionBox.tsx

```diff
@@ -432,7 +439,7 @@ class CompositionBoxInner extends Component<Props, State> {
         {typingEnabled && (
           <ToggleEmojiButton ref={this.emojiPanelButton} onClick={this.toggleEmojiPanel} />
         )}
-        {typingEnabled && <SendMessageButton onClick={this.onSendMessage} />}
+        {showSendButton && <SendMessageButton onClick={this.onSendMessage} />}
```

Comment:
@Bilb We need to fix the spacing / alignment of the emoji button. It is left aligned and the position is off and not symmetrical. Ryan has suggested we use flex gap instead of margin.

<img width="1380" alt="Screenshot 2024-09-06 at 16 08 41" src="https://github.com/user-attachments/assets/3820094a-aac2-4cb1-b6e6-3659522dda55">

Reply:
not a string issue though, backlog?

Reply:
ok, created a ticket SES-2711